Tests/UIKitTests/UIImageExtensionsTests.swift

```diff
@@ -47,6 +47,16 @@ final class UIImageExtensionsTests: XCTestCase {
         XCTAssertNil(UIImage().compressed())
     }
 
+        func testCompressedData() {
```

Comment:
Nit: Identation

Reply:
ohh.. I thought SwiftLint would catch stuff like that.. 🤔